### PR TITLE
Updated Terminal theme and a new Discord theme (this is the last one about themes i promise)

### PR DESCRIPTION
**Pull request by DevMarlin**
Basically Terminal theme before:

image

and after:

image

I wanted to make this update for it to not be that annoying when reading wikis etc. Also made it more accurate to an actual Command Prompt in Windows.

And a new theme: Discord

image

Basically this was created to people that like Discord dark mode a lot!

### CODE REVIEW

File: src/theme/css/variables.css [hunk 1]
```diff
@@ -334,10 +334,10 @@
 
 }
 
-.terminal {
-    font-family: Inconsolata, monospace;
-    text-shadow: 0 0 10px #149414;
+.terminal a:not(.header) {
+    text-shadow: 0 0 10px#00ff05;
 }
+
```
Comment (priyanujgogoi-28) [inline src/theme/css/variables.css]:
```suggestion
```

File: src/theme/css/variables.css [hunk 2]
```diff
@@ -334,10 +334,10 @@
```
Comment (priyanujgogoi-28) [inline src/theme/css/variables.css]:
```suggestion
```

**Review by priyanujgogoi-28 — APPROVED** [review]
Thank you for your contribution!

**Review by MineBartekSA — APPROVED** [review]
Thanks for your contribution.
I'm saying it here, this is the last theme we are accepting without a really good reason.
We have too many themes

**Pull request merged**